tests/monitor.cpp

```diff
@@ -124,12 +124,36 @@ TEST_CASE("monitor init get event count", "[monitor]")
     CHECK_FALSE(monitor.get_event(eventMsg, address, zmq::recv_flags::dontwait));
     s.init();
 
-    while (total < expected_event_count)
+    SECTION("get_event")
     {
-        if (!monitor.get_event(eventMsg, address))
-            continue;
+        while (total < expected_event_count)
+        {
+            if (!monitor.get_event(eventMsg, address))
+                continue;
+
+            lbd_count_event(eventMsg);
+        }
+
+    }
+
+    SECTION("poll get_event")
+    {
+        while (total < expected_event_count)
+        {
+            zmq::pollitem_t items[] = {
+                { monitor.handle(), 0, ZMQ_POLLIN, 0 },
+            };
+
+            zmq::poll(&items[0], 1, 100);
```

Comment:
Please use the overload taking `std::chrono::milliseconds`.

Reply:
Roger that